examples/typescript-reactflux/.gitignore

```diff
@@ -0,0 +1,19 @@
+# Generated folders 
+build/
+node_modules/
```

Comment:
@arthurvr  I heavily use `node_modules` but only for the build chain. That's why I gitignore it. Most repos do that. I see other examples checking in files from `node_modules` but I consider that bad style.